src/KimeraVioRos.cpp

```diff
@@ -10,16 +10,16 @@
 #include <glog/logging.h>
 
 // Dependencies from ROS
-#include <ros/ros.h>
+#include "rclcpp/rclcpp.hpp"
 
 // Dependencies from VIO
 #include <kimera-vio/pipeline/Pipeline.h>
 #include <kimera-vio/utils/Timer.h>
 
 // Dependencies from this repository
-#include "kimera_vio_ros/RosBagDataProvider.h"
-#include "kimera_vio_ros/RosDataProviderInterface.h"
-#include "kimera_vio_ros/RosOnlineDataProvider.h"
+// #include "kimera_vio_ros/RosBagDataProvider.hpp"
+#include "kimera_vio_ros/RosDataProviderInterface.hpp"
```

Comment:
Prefer `.h`, just a matter of code-style.

Reply:
Any reason for not making a distinction between C and C++ headers? I come from an embedded  background, so I never liked conflating the two file extensions. Also makes it harder for linters, IDEs, and humans to disambiguate: https://stackoverflow.com/a/152671

Reply:
I see, we used `.h` from the early beginning, so it's more like legacy for us, and since we don't use C, it's never been a problem so far...
Feel free to keep the .hpp, I might consider refactoring all headers.